.github/workflows/pull-request-test.yml

```diff
@@ -110,34 +112,25 @@ jobs:
     - name: Test the PR
       if: needs.check-deploy.outputs.pr-contains-string == 'true'
       env:
-        RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
+        KUBECONFIG: ${{ github.workspace }}/renkubot-kube.config
+        RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
+        RENKU_RELEASE: ci-renku-${{ github.event.number }}
       run: |
-        export PR_NUMBER=${{ github.event.number }}
-        echo $PR_NUMBER
-        export RENKU_RELEASE="ci-renku-${PR_NUMBER}"
-
-        RENKU_PYTHON_VERSION=v$(cat ./helm-chart/renku/requirements.yaml | yq -r '.dependencies[] | select(.name == "renku-core") | .version')
-        RENKU_PYTHON_COMMIT=$(echo $RENKU_PYTHON_VERSION | grep -o -P '(?<=-)[0-9a-fA-F]+') || true
-        if [ $RENKU_PYTHON_COMMIT ]; then export RENKU_PYTHON_VERSION=$RENKU_PYTHON_COMMIT; fi
-        echo "Passing renku-python version $RENKU_PYTHON_VERSION"
-
-        cd acceptance-tests
-        COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build --build-arg renku_python_ref=${RENKU_PYTHON_VERSION} sbt
-        docker-compose run -e RENKU_TEST_URL=https://${RENKU_RELEASE}.dev.renku.ch \
-                           -e RENKU_TEST_FULL_NAME="Renku Bot" \
-                           -e RENKU_TEST_EMAIL="renku@datascience.ch" \
-                           -e RENKU_TEST_REGISTER="1" \
-                           -e RENKU_TEST_USERNAME="renku-test" \
-                           -e RENKU_TEST_PASSWORD="$RENKU_BOT_DEV_PASSWORD" \
-                           -e RENKU_TEST_ANON_AVAILABLE="true" sbt
-    - name: Prepare artifact for packaging on failure
+        echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
+        helm test ${RENKU_RELEASE} --namespace ${RENKU_RELEASE} --timeout 40m --logs
+    - name: Download artifact for packaging on failure
```

Comment:
Would it be possible to refactor this into a reusable [composite steps action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action)? See [here](https://github.com/SwissDataScienceCenter/renku/blob/master/actions/check-pr-description/action.yaml) for an example of one of our own. Then we can just call it from all the repos to get screenshots on failure. This could also be done as a follow-up issue.